tests/python/test_generate_textlint_runtime.py

```
@@ -0,0 +1,91 @@
+
+    written = json.loads(dest.read_text(encoding="utf-8"))
+    assert "prh" not in written["rules"]
+    assert written["rules"]["other-rule"] is True
+
+
+def test_prh_unsupported_type_raises_type_error(tmp_path):
+    src = _write(tmp_path / "src.json", {"rules": {"prh": "string-not-allowed"}})
+    prh = _make_prh(tmp_path)
+    dest = tmp_path / "runtime.json"
+
+    with pytest.raises(TypeError, match="rules.prh"):
```

Ruff (0.15.12)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)